app/uk/gov/hmrc/exports/controllers/HeaderValidator.scala

```diff
@@ -27,8 +27,8 @@ class HeaderValidator {
   def extractLrnHeader(headers: Map[String, String]): Option[String] =
     extractHeader(CustomsHeaderNames.XLrnHeaderName, headers)
 
-  def extractDucrHeader(headers: Map[String, String]): Option[String] =
-    extractHeader(CustomsHeaderNames.XDucrHeaderName, headers)
+  def extractOptionalDucrHeader(headers: Map[String, String]): Option[String] =
+    headers.get(CustomsHeaderNames.XDucrHeaderName)
```

Comment:
Why are you come back to version with `headers.get` instead of `extractHeader`?

Reply:
As its optional header so no need to print out error in log file if it's missing.